third_party/gems/setup_bundler.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# A template script to unpack the internals of the bundler gem into a local tree
+# that looks like the site_ruby tree that the tracepoint tracer is expecting to
+# find
+
+set -euo pipefail
+
+cd "%{bundler}"
```

Comment:
What is this `%{...}` syntax?

Reply:
It's a template that's replaced during the repository rule. I'm happy to pick a different word, but this seemed like an easy way to pick substitution strings. Here's a use of `repo_ctx.template` where substitutions are defined:
https://github.com/sorbet/sorbet/pull/939/files#diff-d6d62f775ac82d0f76fbca30cb412757R164

Reply:
Ah interesting I didn't realize we got to choose the substitution characters.

I might have gone with `{{bundler}}` myself because that's what a lot of other templating libraries use, but that's just my thoughts and not blocking.

Reply:
I'm happy to change this.

Reply:
I've changed them to `{{name}}`.